scanline/ScanlineAppController.swift

```diff
@@ -276,7 +284,7 @@ class ScanlineOutputProcessor {
     }
     
     func process() -> Bool {
-        if configuration.config[ScanlineConfigOptionJPEG] != nil {
+        if configuration.config[ScanlineConfigOptionJPEG] != nil || configuration.config[ScanlineConfigOptionTIFF] != nil {
```

Comment:
Just to make this a little more clear, and perhaps future proof a bit
```suggestion
        let wantsPDF = configuration.config[ScanlineConfigOptionJPEG] == nil && configuration.config[ScanlineConfigOptionTIFF] == nil
        if !wantsPDF {
```
(just to make things a little more clear and perhaps future proof)

Reply:
I definitely agree that ScanConfiguration could benefit from some improvements. One of the things I tested for the changes was how it reacted when both -tiff and -jpeg were given, as there wasn't a built-in way to say only allow one value out of several.

I think the wantsPDF change makes sense. I just pushed it up to my branch.